### DISCUSSION TITLE
DataTable does not get refreshed

### DISCUSSION THREAD

**Discussion by schnurbus** [huntabyte/shadcn-svelte#1274] [Help] [answered]
Hi,

I try to implement a DataTable like it is described on the Docs page. The only exception to the provided code is, that I send the data to the data-table.svelte via props. The data is fetched in the load function in the +page.server.ts file from the database. If I add an entry via a form on the same page, the data table does not get refreshed. If I render the data directly on the page, the data get refreshed immediately. Could someone gimme a hint, what I'm doing wrong?

Thanks.

**Comment by shyakadavis** [comment, marked as answer]
Hello, @Schnurbus 

This should help, https://github.com/bryanmylee/svelte-headless-table/discussions/105#discussioncomment-4928758

Cheers.

**Comment by schnurbus** [comment]
Hi @shyakadavis 

Thank you very much for the really quick reply. This helps alot.
